### ISSUE TITLE
Mission control click doesn't work when require mouse click enabled

### ISSUE THREAD

**Issue by dvlogic** [pablopunk/SwiftShift#77]
### Bug description

When I use mission control to see all open windows, I am unable to click on a window to select it. This doesn't work ONLY if I have the "require mouse click enabled" for the SwiftShift features.  With the require mouse click NOT enabled, mission control works as normal.

### Setup

<img width="366" alt="image" src="https://github.com/user-attachments/assets/9bf44c51-f4af-47cc-aef8-ab6b528a6c22" />

#### macOS version

Sequoia 15.2 (24C101)

#### Settings

- [ ] Launch at login
- [x] Show menubar icon
- [x] Focus on window
- [x] Use quadrants
- [x] Require mouse click

#### Shortcuts

Move: cmd+alt+opt
Click: left

Resize: cmd+alt+opt
Click: right

**Comment by pablopunk** [comment]
@dvlogic that's odd, I've tried your setup and I didn't get any issues. Can you make a video of it? Maybe I'm missing something

https://github.com/user-attachments/assets/2fcbb298-4ea7-481b-b64c-4fd585902cb9

**Comment by dvlogic** [comment]
Here is my screen capture - its interesting even quicktime recording the screen doesn't show the mouse clicks when the click to move is enabled.   The first 12-seconds of the video show me trying to select windows from mission control.  The next 12 or so seconds shows I'm able to select windows when the left click for move is None.  Finally I turn move with left click back on and again can't select the windows.

https://github.com/user-attachments/assets/9d37e7af-fa69-433d-a703-299c0868f381

**Comment by dvogel-fp** [comment]
Yes still is not allowing me to click on a Mission Control window to select.  I did just notice if I use the Right click or None for resize, then it works as expected - it only doesn't work with I use left click for both Move and Resize.

<img width="366" alt="Image" src="https://github.com/user-attachments/assets/764df548-9ec6-458d-95db-b0e1bd146e99" />

**Comment by jorgebef** [comment]
After using this app for a while, I've found this issue too and narrowed a couple of things further:

I believe the issue could come from some sort of interception from Mission Control when using the same mod keys as SwiftShift, for example:
If I have the following settings for SwiftShift:
<img width="323" alt="Image" src="https://github.com/user-attachments/assets/b2fcd81e-3b36-4fb8-9353-b021b6d50562" />

 If I set the Mission Control keyboard shortcut to something like `Ctrl+Cmd+[anything]` it won't accept clicks *and* the move function sticks, meaning that clicks anywhere won't work as expected, and when I do click, if I drag it will move the window.
So basically it detects the key combo, it gets intercepted by Mission Control, and once you exit Mission Control it still waits for the click even if you have released the modifier keys.

However, if I set the Mission Control keyboard shortcut to anything other that doesn't collide with SwiftShift, for example for my case, `Ctrl+up`, it will work without issue.

Additionally, if you trigger Mission Control without the use of the keyboard shortcut (for example, calling it from Raycast) everything works as expected.

Therefore, I believe the issue is not Mission Control itself, but rather the keyboard shortcut and how it intercepts SwiftShift's ability to read the click or how SwiftShift waits for said click.

Unfortunately, I have 0 experience in Swift codebases, but I will nonetheless take a look at it and hopefully be able to locate something.

I am more than happy to provide any additional info!

Thank you for this great project!

**Comment by pablopunk** [comment]
Released a new version: 1.2.0
